pkg/proxy/iptables/proxier.go

```diff
@@ -682,7 +683,7 @@ func (proxier *Proxier) deleteEndpointConnections(connectionMap []proxy.ServiceE
 	for _, epSvcPair := range connectionMap {
 		if svcInfo, ok := proxier.serviceMap[epSvcPair.ServicePortName]; ok && svcInfo.Protocol() == api.ProtocolUDP {
 			endpointIP := utilproxy.IPPart(epSvcPair.Endpoint)
-			err := utilproxy.ClearUDPConntrackForPeers(proxier.exec, svcInfo.ClusterIP(), endpointIP)
+			err := conntrack.ClearEntryForIPPair(proxier.exec, svcInfo.ClusterIP(), endpointIP, v1.ProtocolUDP)
```

Comment:
IPPair is not quite descriptive - I would assume it is client/dest pair but it's really a NAT record.  ClearEntriesForNAT() ?